src/main/java/seedu/storemando/logic/commands/ReminderCommand.java

```diff
@@ -26,9 +28,12 @@ public ReminderCommand(ItemExpiringPredicate predicate) {
     }
 
     @Override
-    public CommandResult execute(Model model) {
+    public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
         model.updateFilteredItemList(predicate);
+        ItemComparatorByExpiryDate comparator = new ItemComparatorByExpiryDate();
+        model.updateSortedItemList(comparator);
+        model.setItems(model.getSortedItemList());
```

Comment:
Can we abstract it out further? This method calls setItems method of model class and retrieves model's own attribute to pass back as a parameter. I think we can create a method to abstract this out further.

Reply:
I will create a method inside model :)

Reply:
I think this is fine because I'm calling the itemlist in model itself and not somewhere else right :)